frontend/public/components/graphs/area.tsx

```diff
@@ -55,7 +55,8 @@ export const AreaChart: React.FC<AreaChartProps> = ({
   query,
   tickCount = DEFAULT_TICK_COUNT,
   title,
-  ariaChartLabel,
+  showTitle = true,
```

Comment:
It's confusing to have both `showTitle` and `title` properties. I would expect that you leave `title` undefined if you don't want it to display. We need to give this prop a better name. If it's only for accessibility, then we should make that clear in the prop name.

Reply:
Ok, removed `showTitle` and added `ariaChartTitle`.  If `title` is defined, it will be used as the a11y `<title ...` of the svg chart image:
![image](https://user-images.githubusercontent.com/12733153/104958669-722a1f00-599e-11eb-893e-29f86bbf71ac.png)

if `title` isn't defined, then user can set `ariaChartTitle` which will be used in the the a11y `<title ...` of the svg chart image (note Chart doesn't have a visible title):

![image](https://user-images.githubusercontent.com/12733153/104958760-9ab21900-599e-11eb-9732-530a015a4e2e.png)